wsclient/src/test/java/org/hpccsystems/ws/client/BaseRemoteTest.java

```diff
@@ -85,6 +86,32 @@ public abstract class BaseRemoteTest
     public static final String DEFAULTHPCCSUPERFILENAME = "benchmark::all_types::superfile";
 
     static
+    {
+        try
+        {
+            initialize();
+            initialized = true;
```

Comment:
not a big deal, but I wonder if the initialized bool is necessary. It seems initializationException can be used to determine if init succeeded/failed.